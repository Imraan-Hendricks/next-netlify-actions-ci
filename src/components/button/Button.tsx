import { PropsWithChildren } from 'react';

export function Button({ children }: PropsWithChildren) {
  return (
    <button className='bg-teal-400 hover:bg-teal-500 active:bg-teal-400 text-white py-2.5 px-7 rounded transition-colors'>
      {children}
    </button>
  );
}
