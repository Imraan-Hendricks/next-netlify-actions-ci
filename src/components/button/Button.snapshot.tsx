import { Button } from './Button';
import { render } from '@testing-library/react';

describe('Button snapshot tests', () => {
  it('renders with expected children', () => {
    const { container } = render(<Button>Click Me</Button>);
    expect(container).toMatchSnapshot();
  });
});
