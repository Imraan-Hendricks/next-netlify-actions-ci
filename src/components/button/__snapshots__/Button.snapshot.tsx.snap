// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button snapshot tests renders with expected children 1`] = `
<div>
  <button
    class="bg-teal-400 hover:bg-teal-500 active:bg-teal-400 text-white py-2.5 px-7 rounded transition-colors"
  >
    Click Me
  </button>
</div>
`;
